### PR TITLE
[feat] Adds create page button to pages list

### PR DESCRIPTION
**Pull request by marcoviappiani**
- adds create page button in the pages list

### CODE REVIEW

File: modules/core/client/controllers/pagesAdminList.controller.client.js [hunk 1]
```diff
@@ -36,5 +41,6 @@ angular.module('tropicalbs')
   // TODO: ideally we should decide to show a snippet of the content or not show the content at all
   Pages.getAllPages().then(function(res){
     $scope.allPages = res;
+    console.log(res);
```
Comment (codydaig) [inline modules/core/client/controllers/pagesAdminList.controller.client.js]: console.log

File: modules/core/client/css/core.login.css [hunk 2]
```diff
@@ -2,3 +2,10 @@
 /* CSS FOR ANGULAR MATERIAL
 /**************************/
 
+.md-button.md-fab.md-fab-bottom-right {
+top: auto;
```
Comment (codydaig) [inline modules/core/client/css/core.login.css]: indentation

File: modules/core/client/views/page-admin-list.view.client.html [hunk 3]
```diff
@@ -16,3 +19,4 @@
     </md-list-item>
   </md-list>
 </md-content>
+
```
Comment (codydaig) [inline modules/core/client/views/page-admin-list.view.client.html]: Extra New Line

**Comment by RRDuke** [comment]
LGTM

**Comment by marcoviappiani** [comment]
thanks for the comments! I included them into this new pull request

**Pull request merged**